src/components/page/Header/index.tsx

```diff
@@ -41,6 +43,15 @@ export const HeaderComponent: React.FC<HeaderProps> = (props: HeaderProps) => {
         </div>
         <Menu mode="horizontal" selectedKeys={[path]}>
           <Menu.Item key={HOME_URL}>Home</Menu.Item>
+
+          <SubMenu title="Language" style={{ float: 'right' }}>
```

Comment:
I think that we do not yet have a requirement for a language switcher.  So far we've been deploying the web app in one language that is chosen/set during deployment.   @dubdabasoduba @ukanga could you chime in please?

Reply:
ok for now i am removing them will add them again if required

Reply:
We are deploying with on language but a couple of implementations are already asking for this.  Is there a way to make it that the language switcher can be toggled on/off?

Reply:
yeah we can add a variable in environment file to change that behavior

Reply:
Just Added language switcher with environment variable